libsolidity/codegen/ir/IRGenerator.cpp

```diff
@@ -617,15 +635,17 @@ string IRGenerator::generateGetter(VariableDeclaration const& _varDecl)
 		}
 
 		return Whiskers(R"(
-			function <functionName>(<params>) -> <retVariables> {
-				<code>
-			}
-		)")
-		("functionName", functionName)
-		("params", joinHumanReadable(parameters))
-		("retVariables", joinHumanReadable(returnVariables))
-		("code", std::move(code))
-		.render();
+				<sourceLocationComment>
+				function <functionName>(<params>) -> <retVariables> {
+					<code>
+				}
+			)")
```

Comment:
The line with the closing parenthesis is indented exactly as far as the line containing the open parenthesis.

Reply:
This was an accident, I never meant to change any indentation here

Reply:
But now I understand why it showed everything as changed and not just my additions. I was confused by that earlier